components/dataproducts/editDataproduct.tsx

```diff
@@ -29,14 +28,16 @@ const EditDataproduct = ({ product }: EditDatacollectionFormProps) => {
         description: product.description || '',
         teamkatalogenURL: product.owner.teamkatalogenURL,
         teamContact: product.owner.teamContact,
+        productAreaId: product.owner.productAreaId,
       },
     })
+  const [productAreaId, setProductAreaId] = useState<string>('')
 
   const { errors } = formState
-  const onSubmit = (requestData: UpdateDataproduct) => {
-      
+  const onSubmit = (requestData: any) => {
```

Comment:
If you run codegen, the UpdateDataproduct model is updated with the productAreaId prop. Then you won't have to type it to any here.

Reply:
I see!